examples/ml-slurm-v6.yaml

```diff
@@ -0,0 +1,267 @@
+                  exit 0
+          fi
+
+          DL_DIR=\$(mktemp -d)
+          cd $DL_DIR
+          curl -O https://repo.anaconda.com/miniconda/Miniconda3-py310_23.3.1-0-Linux-x86_64.sh
+          HOME=$DL_DIR bash Miniconda3-py310_23.3.1-0-Linux-x86_64.sh -b -p $CONDA_BASE
+          cd -
+          rm -rf $DL_DIR
+          unset DL_DIR
+
+          source $CONDA_BASE/bin/activate base
+          conda init --system
+          conda config --system --set auto_activate_base False
+          # following channel ordering is important! use strict_priority!
+          conda config --system --set channel_priority strict
+          conda config --system --remove channels defaults
+          conda config --system --add channels conda-forge
+          conda config --system --add channels nvidia
+          conda config --system --add channels nvidia/label/cuda-11.8.0
+
+          conda update -n base conda --yes
+
+          ### create a virtual environment for tensorflow
+          conda create -n tf python=3.10 --yes
+          conda activate tf
+          conda install -n tf cuda-toolkit --yes
+          pip install nvidia-cudnn-cu11 nvidia-nccl-cu11
+
+          cd $CONDA_PREFIX/lib/python3.10/site-packages/nvidia/nccl/lib/
+          ln -s libnccl.so.2 libnccl.so
+          cd -
+
+          mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+          echo 'export OLD_LD_LIBRARY_PATH=$LD_LIBRARY_PATH' > $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+          echo 'NVIDIA_PYTHON_PATH=$CONDA_PREFIX/lib/python3.10/site-packages/nvidia' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+          echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/:$NVIDIA_PYTHON_PATH/cudnn/lib/:$NVIDIA_PYTHON_PATH/nccl/lib/' >> $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh
+          mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d
+          echo 'export LD_LIBRARY_PATH=${OLD_LD_LIBRARY_PATH}' > $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+          echo 'unset OLD_LD_LIBRARY_PATH' >> $CONDA_PREFIX/etc/conda/deactivate.d/env_vars.sh
+
+          pip install tensorflow==2.12.*
+          pip install tensorrt==8.6.*
+
+          ### create a virtual environment for pytorch
+          conda create -n pytorch python=3.10 --yes
+          conda activate pytorch
+          conda config --env --add channels pytorch
+          conda install -n pytorch pytorch torchvision torchaudio pytorch-cuda=11.8 --yes
+
+- group: packer
+  modules:
+  - id: custom-image
+    source: modules/packer/custom-image
+    kind: packer
+    use:
+    - network
+    - script
+    settings:
+      # give VM a public IP to ensure startup script can reach public internet
+      # w/o new VPC
+      omit_external_ip: false
+      source_image_project_id: [schedmd-slurm-public]
+      # see latest in https://github.com/GoogleCloudPlatform/slurm-gcp/blob/master/docs/images.md#published-image-family
+      source_image_family: slurm-gcp-6-4-hpc-rocky-linux-8
```

Comment:
The v5 version uses Debian as the base image. I think that is kind of important to the example. Why has it changed?

Reply:
PS. If we use Debian then this example would be one to add to the support matrix for slurm + startup script + debian

Reply:
No specific reason other than its default for SlurmGCP v6. I will update it to use debian.

Reply:
Updated to debian. For matrix I created new task and will update for everything at once.